dockerfiles/vcpkg/centos-7.dockerfile

```diff
@@ -1,9 +1,22 @@
 ARG VCPKG_REF=latest
-FROM hpccbuilds/vcpkg-centos-7:$VCPKG_REF
+ARG DOCKER_NAMESPACE=hpccbuilds
+FROM ${DOCKER_NAMESPACE}/vcpkg-centos-7:$VCPKG_REF
+
+ENV Rcpp_package=Rcpp_0.12.19.tar.gz
+ENV RInside_package=RInside_0.2.12.tar.gz
```

Comment:
Move closer to where they are referenced (after the yum install stuff)

Reply:
Moved to just above the fetches with wget.